index.js

```diff
@@ -597,7 +597,14 @@ async function verifyRevision(
   // TODO do sanity check on domain id
   const domainId = data.metadata.domain_id
 
-  const contentHash = getHashSum(data.content.content)
+  let content = ''
+  for ( var slot in data.content.content ) {
```

Comment:
I think you can do instead:
```js
for (const [slot, slotContent] of Object.entries(data.content.content_) {
```
? Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries.
Then you no longer need to do the `hasOwnProperty`.
> This is the same as iterating with a for...in loop, except that a for...in loop enumerates properties in the prototype chain as well.

Also, you are using PHP code style here, with the spaces after and before the parentheses.